### PR TITLE
Voice audition

### PR DESCRIPTION
**Pull request by charli117**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] TODO

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

### CODE REVIEW

File: web/utils/language.ts [hunk 1]
```diff
@@ -10,54 +11,67 @@ export const languages = [
...
   },
   {
     value: 'de-DE',
     name: 'Deutsch(Deutschland)',
+    example: 'Hallo, Dify!',
   },
   {
     value: 'ja-JP',
     name: '日本語(日本)',
+    example: 'こんにちは、ディファイ!',
```
Comment (crazywoola) [inline web/utils/language.ts]: I think we can keep `Dify` for none English languages and keep them as Capitalized.

**Review by crazywoola — CHANGES_REQUESTED** [review]
https://github.com/langgenius/dify/blob/1b2fe64fe10d34606ce32f17c27cd7b12e114e88/web/app/components/base/audio-btn/index.tsx#L45

I think it's better to move these into services as well.

File: web/utils/language.ts (hunk 1 above)
Comment (charli117) [reply web/utils/language.ts]: > I think we can keep `Dify` for none English languages and keep them as Capitalized.

example has been adjusted

File: web/utils/language.ts [hunk 2]
```diff
@@ -10,54 +11,67 @@ export const languages = [
...
   },
   {
     value: 'es-ES',
     name: 'Español(España)',
+    example: 'Saluton, Dify!',
   },
   {
     value: 'fr-FR',
     name: 'Français(France)',
+    example: 'Bonjour, dify!',
```
Comment (crazywoola) [inline web/utils/language.ts]:
```suggestion
    example: 'Bonjour, Dify!',
```

**Review by crazywoola — DISMISSED** [review]

**Review by crazywoola — APPROVED** [review]

**Pull request merged**